src/website/views/components/toolbar/_examples.pug

```diff
@@ -1,499 +1,1371 @@
-                  <div class="chi-button__content">
-                    <i class="chi-icon icon-arrow-to-bottom"></i>
+        </div>
+
+        <!-- Javascript -->
+        <script>
+          chi.drawer(document.getElementById('drawer-trigger-br1'));
+          chi.drawer(document.getElementById('drawer-trigger-br2'));
+          chi.tooltip(document.querySelectorAll('[data-tooltip]'));
+        </script>
+
+.-portal--show
+  h3 Portal base
+  .example.-mb--3
+    .-p--3
+      .chi-toolbar.-portal
+        .chi-toolbar__header
+          .chi-toolbar__start
+            .chi-toolbar__search
+              .chi-form__item
+                chi-search-input(portal size="lg" id="example__base-3")
+            .chi-divider.-vertical
+            .chi-toolbar__filters
+              .chi-toolbar__filters-desktop
+                .chi-form__item
+                  select(class="chi-select -lg -portal", id="example-ba5")
+                    option(value="") Select
+                    option Option 1
+                    option Option 2
+                    option Option 3
+                .chi-form__item
+                  select(class="chi-select -lg -portal", id="example-ba6")
+                    option(value="") Select
+                    option Option 1
+                    option Option 2
+                    option Option 3
+                button.chi-button.-portal.-icon.-primary.-flat(id="button-p-filters" aria-label="Filters" data-tooltip="Filters" data-position="top")
+                  .chi-button__content
+                    i.chi-icon.icon-filter
+              .chi-toolbar__filters-mobile
+                button#drawer-trigger-filter1.chi-button.-portal.-icon.-primary.-flat.chi-drawer__trigger(data-target='#drawer-1', aria-label="Button action")
+                  .chi-button__content
+                    i.chi-icon.icon-filter
+          .chi-toolbar__end
+            .chi-toolbar__actions
+              .chi-toolbar__actions-desktop
+                button.chi-button.-portal.-icon.-primary.-flat(id="button-p-download" aria-label="Download" data-tooltip="Download" data-position="top")
+                  .chi-button__content
+                    i.chi-icon.icon-arrow-to-bottom
+                button.chi-button.-portal.-icon.-primary.-flat(id="button-p-refresh" aria-label="Refresh" data-tooltip="Refresh" data-position="top")
+                  .chi-button__content
+                    i.chi-icon.icon-refresh
+                button.chi-button.-portal.-icon.-primary.-flat(id="button-p-column-customization" aria-label="Column Customization" data-tooltip="Column Customization" data-position="top")
+                  .chi-button__content
+                    i.chi-icon.icon-table-column-settings
+              .chi-toolbar__actions-mobile
+                button#drawer-trigger-actions1.chi-button.-portal.-icon.-primary.-flat.chi-drawer__trigger(data-target='#drawer-2', aria-label="Button action")
```

Comment:
-portal class is not needed anymmore, please remove

Reply:
Solved.